lib/functions.sh

```diff
@@ -169,6 +169,17 @@ function retry {
   return 0
 }
 
+function bk_is_running {
+	detectCompose
+	RUNNING="$($COMPOSE -p bridgehead-$PROJECT -f ./$PROJECT/docker-compose.yml $OVERRIDE ps -q)"
+	NUMBEROFRUNNING=$(echo "$RUNNING" | wc -l)
+	if [ $NUMBEROFRUNNING -ge 2 ]; then
```

Comment:
why greater equals than 2? Shouldn't we also accept 1 running service?

Reply:
That's the way `wc -l` works -- and there are no Bridgeheads with <2 services